lib/components/viewers/live-stop-times.tsx

```diff
@@ -144,36 +150,88 @@ class LiveStopTimes extends Component<Props, State> {
       return stopTimeComparator(stopTimesA[0], stopTimesB[0])
     }
 
+    const routeTimes = Object.values(stopTimesByPattern)
+      .filter(({ times }) => times.length !== 0)
+      .sort(patternComparator)
+      .map((route) => {
+        return {
+          ...route,
+          times: route.times.concat()?.sort(stopTimeComparator)
+        }
+      })
+      .filter(({ pattern, route }) =>
+        routeIsValid(route, getRouteIdForPattern(pattern))
+      )
+
+    const routesByDay: any = []
+    const firstDayOfService = routeTimes[0].times[0].serviceDay
```

Comment:
```suggestion
    const firstDayOfService = routeTimes?.[0].times?.[0].serviceDay
```

We can't rely on this existing!
<img width="612" alt="image" src="https://user-images.githubusercontent.com/86619099/220986794-ebc7b405-a16e-4c80-9b79-0f6b1ae98ec8.png">